app/src/controllers/messagesController.js

```diff
@@ -31,23 +32,23 @@ export default class MessagesController {
    * @param res
    */
   getUserMessages(req: express$Request, res: express$Response) {
-    const reqWithUser = ((req: Object): { user: User });
+    const user = extractUserFromRequest(req);
+
+    if (user.fiscal_code === undefined) {
+      res.status(500).json({
+        message: "There was an error in retrieving the user profile."
+      });
+      return;
+    }
 
     this.apiClient
-      .getClient(reqWithUser.user.fiscal_code)
+      .getClient(user.fiscal_code)
       .getMessagesByUser()
       .then(
         function(apiMessages: GetMessagesByUserOKResponse) {
-          let appMessages = [];
-          for (let i = 0; i < apiMessages.pageSize; i++) {
-            const apiMessage = apiMessages.items[i];
-            const appMessage: Message = {
-              id: apiMessage.id,
-              subject: "Lorem ipsum", //apiMessage.content.subject,
-              markdown: "Lorem ipsum" //apiMessage.content.markdown
-            };
-            appMessages.push(appMessage);
-          }
+          const appMessages = apiMessages.items.map(apiMessage => {
```

Comment:
I believe this statement may be simplified to `apiMessages.items.map(toAppMessage)`